lib/interfaces/uncaught.js

```diff
@@ -54,6 +54,7 @@ function handlerSetup(client, config) {
 
     errorHandlerRouter(err, em);
     client.sendError(em, handleProcessExit);
+    setTimeout(handleProcessExit, 2000);
```

Comment:
Could we add a test to ensure that errors are published when an uncaught exception happens (if not tested elsewhere)?